src/util/mod.rs

```diff
@@ -369,6 +369,23 @@ impl<L, R> Either<L, R> {
     }
 }
 
+/// `build_info` returns a tuple of Strings that contains build utc time and commit hash.
+fn build_info() -> (String, String) {
+    let raw = include_str!(concat!(env!("OUT_DIR"), "/build-info.txt"));
```

Comment:
I think using `include_str` is ok too. Since all the content in `build-info.txt` is loaded in compile time.